fuzz_lightyear/fuzzer.py

```diff
@@ -74,18 +76,22 @@ def _fuzz_parameter(
             # TODO: handle `file` type
             # https://swagger.io/docs/specification/2-0/file-upload/
         }
-        strategy = mapping[_type](parameter, required=required)
+        fuzz_fn = mapping[_type]
+        if fuzz_fn in (_fuzz_object, _fuzz_array):
+            strategy = fuzz_fn(parameter, operation_id, required=required)  # type: ignore
+        else:
+            strategy = fuzz_fn(parameter, required=required)  # type: ignore
 
     # NOTE: We don't currently support `nullable` values, so we use `None` as a
     #       proxy to exclude the parameter from the final dictionary.
     if (
         # `name` check is used here as a heuristic to determine whether in
         # recursive call (arrays).
-        parameter.get('name')
-        and not required
+        parameter.get('name') and
+        not required
     ):
-        return st.one_of(st.none(), strategy)
-    return strategy
+        return st.one_of(st.none(), strategy)  # type: ignore
+    return strategy  # type: ignore
```

Comment:
Huh. Do you know what `mypy` is complaining about that requires the use of `# type: ignore` scattered here?

Reply:
yeah, my guess is that mypy thinks that strategy isn't guaranteed since we set it in an if/else statement above